src/Makefile

```diff
@@ -118,6 +118,11 @@ uninstall:
 	test ! -f $(BINDIR)/atdgen.exe || rm $(BINDIR)/atdgen.exe
 	ocamlfind remove atdgen
 
+findlib-install:
+	ocamlfind install atdgen META \
+		 $(MLI) $(CMI) $(CMO) $(CMX) $(CMXS) $(O) \
+			atdgen.cma atdgen.a atdgen.cmxa
+
```

Comment:
I agree that those seem like good targets and your intuition is correct. `install` now has `findlib-install` as a dependent so that it ensures a full install. It could just have `findlib-install` and `bin-install` as dependents and no recipe body.